### PR TITLE
Bump navigation to 3.10.0

### PR DESCRIPTION
**Pull request by jeroach**
https://github.com/BrightspaceUI/navigation/releases/tag/v3.10.0

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2858,7 +2858,7 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#43cd7f9c59bcbf6625aa76b5346026b98b4475d3",
+      "version": "github:BrightspaceHypermediaComponents/activities#3342221132a1b92658e2bc8db7ab8329e6f2374e",
```
Comment (jeroach) [inline package-lock.json]: fyi @maboelsoudd2l

File: package-lock.json [hunk 2]
```diff
@@ -2893,7 +2893,7 @@
       }
     },
     "d2l-activity-alignments": {
-      "version": "github:Brightspace/d2l-activity-alignments#0c02f47e11fea1526432263c5eba330425bd6243",
+      "version": "github:Brightspace/d2l-activity-alignments#94f750e4ae47264a72a9f975a79d03c484530167",
```
Comment (jeroach) [inline package-lock.json]: fyi @djwester

File: package-lock.json [hunk 3]
```diff
@@ -3494,7 +3494,7 @@
       }
     },
     "d2l-rubric": {
-      "version": "github:Brightspace/d2l-rubric#997cc3c39bda43b35a7987a011f1c9d159c6da0b",
+      "version": "github:Brightspace/d2l-rubric#a63004de4a9bf1da10bd858e8f998b10109cc0f6",
```
Comment (jeroach) [inline package-lock.json]: fyi @kevinwang-d2l

**Comment by jeroach** [comment]
> I thought we just we just updated this in BSI?

When adding a new web component, you also need to reference it in `bsi.js` so that it becomes available for use in the LMS.

Also, when bumping the BSI version, it's best to pickup all other _minor_ version changes from other linked repos. And then notify those who did the change (as I did). If we skip this process, weird things can end up happening - remember the issue with BSI that Viktor fixed last week :)

**Comment by maboelsoudd2l** [comment]
> > I thought we just we just updated this in BSI?
> 
> When adding a new web component, you also need to reference it in `bsi.js` so that it becomes available for use in the LMS.
> 
> Also, when bumping the BSI version, it's best to pickup all other _minor_ version changes from other linked repos. And then notify those who did the change (as I did). If we skip this process, weird things can end up happening - remember the issue with BSI that Viktor fixed last week :)

Does this process of picking up other minor version happen automatically?
Also @mtjandra updated the activities version to a newer release in #1512 , wouldn't this PR bump it back?

**Comment by jeroach** [comment]
Mohammed and I chatted offline :)

**Pull request merged**